playbooks/roles/ocp-config/defaults/main/main.yaml

```diff
@@ -10,3 +10,5 @@ enable_local_registry: false
 
 chronyconfig:
    enabled: true
+
+cni_network_provider: OpenshiftSDN
```

Comment:
@pravin-dsilva for 4.7, the default is OpenShiftSDN or OVN ?  For main branch we should set default to 4.7 specific values.

Reply:
@bpradipt the default is OpenshiftSDN for 4.7.